src/components/common/CustomCalendar.tsx

```diff
@@ -109,25 +123,33 @@ const CustomCalendar: FC<CustomCalendarProps> = (props) => {
         />
         )
         } />
+      <CustomToast
+        onHide={() => setShowFutureDateToast(false)}
+        showToast={showFutureDateToast}
+        severity="warn"
+        detail="Please select a date and time on or before the current moment."
+        position="top-center"
+        life={3000}
```

Comment:
Don't keep life.
Let the toast be open until user manually closes it.
It's important that the user doesn't miss the toast.
So let the user themselves close the toast.

Reply:
Removed life